index.html

```diff
@@ -1,189 +1,227 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
     <title>My Portfolio</title>
 </head>
-
 <body>
         <header class="main-header"> 
-            <a class="logo" href="#"><img src="My Logo (1).png" alt=""></a>
-            <nav class="nav-container">
-                <a href="#" class="menu"><img src="menu.png"page logo=""></a>       
+            <a class="logo" href="#"><img src="My Logo (1).png" alt="home page logo"></a>
+            <nav id="nav-container">
+                    <a class="desk" href="#" >Project</a>
+                    <a class="desk" href="#" >About</a> 
+                    <a class="desk" href="#" >Contact</a>
+            <a class="menu" href="#" ><img src="menu.png"alt="menu"></a>    
         </nav>
     </header>
-    <div  class="container">
         <section class="part1">
-        <h1 class="head-name">I’m Mumbere <br>
-            Glad to see you!</h1>
-        <p class="head-para">I’m a software developer! I can help you build a product ,
+        <div>
+        <h1 class="head-name">I’m Mumbere. <br>
+        Glad to see you!</h1>
+        </div>
+        <p class="head-para">
+            I’m a software developer! I can help you build a product ,
             feature or website Look through some of my work and experience!
             If you like what you see and have a project you need coded,
-            don’t hestiate to contact me.</p>
-            <h2 class="sub-head">Let's Connect</h2>
-            <div class="social-media">
+            don’t hestiate to contact me.
+        </p>
+        <div class="sub-head">
+            <h2>Let's Connect</h2>
+        </div>
+            <div class="social-media">    
```

Comment:
- [ ] Firstly, kindly adjust the space between the `header` content text and social icons to be a replica of the design given 👍 
- [ ] Secondly, lets adjust the placemnt of the background image. You can see the botton dots is lacking on your design. These chanages might seem minor to you, but as a developer, you're tasked to replicated the works of poduct designsers, of which they take every detail into consideration. In prepario for clients and your mangers work, kindly align the image text properly 👍 
## Your view:
![image](https://github.com/mumbereh/my-portfolio/assets/50990925/1474f393-fd67-4814-aec4-dce7019a4e3a)

## Figma view:
![image](https://github.com/mumbereh/my-portfolio/assets/50990925/5be65128-302e-4be0-bb01-5e92fd0042a0)